toml/parser.ts

```diff
@@ -539,10 +539,18 @@ export function parse(tomlString: string): object {
   return new Parser(tomlString).parse();
 }
 
-export function parseFile(filePath: string): object {
+export function parseFileSync(filePath: string): object {
   if (!existsSync(filePath)) {
     throw new Error("File not found");
   }
   const strFile = readFileStrSync(filePath);
   return parse(strFile);
 }
+
+export async function parseFile(filePath: string): Promise<object> {
+  if (!(await exists(filePath))) {
+    throw new Error("File not found");
+  }
+  const strFile = await readFileStr(filePath);
+  return parse(strFile);
+}
```

Comment:
Actually I'd rather that we remove the `parseFileSync` and leave it up to the caller to load the data. The TOML module should only deal with parsing, not with I/O.

Would you mind a revised PR that removes `parseFileSync` (or just moves it to the test module - if that is needed there)?

Reply:
That makes a lot of sense to me!